app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,13 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str):
+        super(Dwarf, self).__init__(nickname)

```

Comment:
If you do not need to change the standard arguments, you can not write them. Python 3 will substitute them for you.